File: app/src/main/java/com/niceplaces/niceplaces/utils/StringUtils.java
package com.niceplaces.niceplaces.utils;


import android.util.Log;

import java.util.List;

public class StringUtils {

    public static String listToString(List<Integer> list){
        String string = "";
        for (int i = 0; i < list.size(); i++){
            string = string.concat(Integer.toString(list.get(i)));
        }
        return string;
    }

    public static void printList(List<String> list){
        for (int i = 0; i < list.size(); i++){
            Log.i("STRING_ARRAY", "[" + i + "] " + list.get(i));
        }
    }
}

File: app/src/main/java/com/niceplaces/niceplaces/utils/StringUtils.kt
package com.niceplaces.niceplaces.utils

import android.util.Log
import java.text.SimpleDateFormat
import java.util.*

object StringUtils {
    fun listToString(list: List<Int?>): String {
        var string = ""
        for (i in list.indices) {
            string = string + Integer.toString(list[i]!!)
        }
        return string
    }

    fun printList(list: List<String>) {
        for (i in list.indices) {
            Log.i("STRING_ARRAY", "[" + i + "] " + list[i])
        }
    }

    fun timestampToDate(l: Long?): String? {
        return if (l != null){
            try {
                val sdf = SimpleDateFormat("dd/MM/yyyy", Locale.ROOT)
                val netDate = Date(l)
                sdf.format(netDate)
            } catch (e: Exception) {
                e.toString()
            }
        } else {
            l.toString()
        }
    }
}